ember-aria-utilities/src/test-support/-private/aria-grid.ts

```diff
@@ -105,5 +153,6 @@ const keys = {
 
 export const ariaGrid = {
   keys,
-  selectors,
```

Comment:
this is a change in public API and would require a breaking change be declared

Reply:
Thanks for catching this / pointing this out.

Reply:
Wait...should I put a changeset in this PR then?

Reply:
this repo doesn't use changeset (yet?), but instead you'd want:

```
kind(topic): subject

BREAKING CHANGE: why
```

for the commit message